src/api/article/controller.js

```diff
@@ -42,23 +44,33 @@ export const getAll = ({ query, user }, res, next) => {
     .catch(next);
 };
 
-export const getOne = ({ params: { slug }, user }, res, next) =>
-  Article.findOne({ slug, active: true })
+export const getOne = async ({ params: { slug }, user }, res, next) => {
+  // TODO(uladbohdan): to fix for unexisting article.
+  let articleData;
+  try {
+    articleData = await ArticleData.findOne({ slug }).exec();
```

Comment:
I think with promise chain this method would look more clean

Reply:
Same as below: I don't know how to implement a chain if we work with different models at the same time.

Reply:
```js